MediaBrowser.Model/Dlna/ResolutionNormalizer.cs

```diff
@@ -38,16 +43,26 @@ public static ResolutionOptions Normalize(
                 }
             }
 
-            var resolutionConfig = GetResolutionConfiguration(outputBitrate);
-            if (resolutionConfig is not null)
+            var referenceBitrate = h264EquivalentOutputBitrate * (30.0f / (targetFps ?? 30.0f));
```

Comment:
Can `targetFps` be `0` (due to a bug)?

Reply:
I would say just let it throw and fail in this case. This either means:

- the client is requesting 0fps explicitly

Or

- the source video is probed as 0fps

And either way the pipeline will break more than just resolution normalization.